beets/util/artresizer.py

```diff
@@ -430,7 +464,12 @@ def get_format(self, filepath):
             log.exception("failed to detect image format for {}", filepath)
             return None
 
-    def convert_format(self, source, target, deinterlaced):
+    def convert_format(
+            self,
+            source: AnyStr,
+            target: AnyStr,
+            deinterlaced: bool,
+    ) -> str:
```

Comment:
```suggestion
    ) -> AnyStr:
```

given that this returns either of `source` or `target`.

On second thought: Probably all of these should be `bytes` (again, these are paths in beets internal representation, as being evidenced by the `syspath` call).